templates/config/nginx/default.conf.mustache

```diff
@@ -22,7 +22,7 @@ server {
 		set $skip 1;
 	}
 	# Don't cache URL containing the following segments
-	if ($request_uri ~* "(/wp-admin/|/xmlrpc.php|wp-.*.php|index.php|/feed/|sitemap(_index)?.xml|[a-z0-9_-]+-sitemap([0-9]+)?.xml)") {
+	if ($request_uri ~* "(/wp-admin/|/ee-admin/|/xmlrpc.php|wp-.*.php|index.php|/feed/|sitemap(_index)?.xml|[a-z0-9_-]+-sitemap([0-9]+)?.xml)") {
```

Comment:
We have planned to use it in diff package https://github.com/EasyEngine/admin-tools 
So, I think if we can handle this in that package itself that would be great. So that if admin-tool package is not there then this rule also not apply to the site.

Reply:
Okay. We can add it in admin tools package in the following way. 

We can have a custom custom config location `SITE_ROOT/config/nginx/custom`. If you are placing a `rules.conf` file with
```
	if ($request_uri ~* "(/ee-admin/)") {
		set $skip 1;
}```

It would work then from the admin-tools package.